### ISSUE TITLE
Timer not resetting when pressing "restore".

### ISSUE THREAD

**Issue by denispeplin** [Microflow/citybuilder#154]
After pressing "delete", "restore" button appears. But when pressing "restore" timer not being resetting, so after multiple clicks time is no longer 10 seconds, but less.

**Comment by Microflow** [comment]
We can hopefully get help on this from a JS junior after the project is more well known

**Comment by denispeplin** [comment]
I don't think this is a big problem, but it requires to build a logic timer more carefully.
